### DISCUSSION TITLE
Is there a hack to cause high ping on other players?

### DISCUSSION THREAD

**Discussion by diekunstderfuge** [notgoodusename/OsirisAndExtra#541] [Q&A] [answered]
I've seen a guy saying he was going to increase the ping of another player and it happened and it kept that way... And he did the same thing with other player in another match.

Is that really possible? How can I do it?

**Comment by notgoodusename** [comment]
> instead of discussing stuff here, u should go to  https://unknowncheats.me
> im sure they would help u faster or answer ur questions

unknowncheats doesn't answer this kind of questions due to legal reasons, because this type of exploit usually can crash a server, or u can just lag the entire server till it's unplayable, and making it public would also be idiotic
this types of exploit take years to become public (in most cases)

**Comment by notgoodusename** [comment, marked as answer]
> I've seen a guy saying he was going to increase the ping of another player and it happened and it kept that way... And he did the same thing with other player in another match.
> 
> Is that really possible? How can I do it?

more than likely it is possible but you won't find an answer
i advise to look at harpoon src, it still has many server crashers that could work(idk), or you can learn from it

**Reply by diekunstderfuge** [reply]
I managed to contact the guy and he told me he is using Nixware. It currently has a working server crasher.

**Reply by notgoodusename** [reply]
> I managed to contact the guy and he told me he is using Nixware. It currently has a working server crasher.

Yeah it seems as nixware has a server crasher, u could probably get the dll and reverse engineer it

**Comment by diekunstderfuge** [comment]
I hope this crash server gets patched soon. I'm easily tapping all the nixware guys, but they crash the server after I kill them in the last round.
